_posts/2020-11-03-torch-tabular/torch_tabular.Rmd

```diff
@@ -0,0 +1,445 @@
+---
+title: "torch for tabular data"
+description: > 
+  How not to die from poisonous mushrooms. Also: How to use torch for deep learning on tabular data, including a mix of categorical and numerical features.
+
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatorchtabular
+date: 11-03-2020
+categories:
+  - Torch
+  - R
+  - Tabular Data
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/preview.jpeg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+Machine learning on image-like data can be many things: Fun (dogs vs. cats), societally useful (medical imaging), or societally harmful (surveillance). In comparison, tabular data -- the bread and butter of data science -- may seem more mundane.
+
+What's more, if you're interested in deep learning (DL) particularly, and looking for the extra benefits to be gotten from big data, big architectures, and big compute, you're much more likely to build an impressive showcase on the former instead of the latter.
+
+So for tabular data, why not just go with random forests, or gradient boosting, or other classical methods? I can think of at least a few reasons to learn about DL for tabular data:
```

Comment:
I feel like there's a word missing in "So for....tabular data" (i.e., For _what-ing_ with tabular data?)

Reply:
I see the point... However I can't really think of anything not too tautological, given the context ("learning from")?
Since it's still not incomprehensible, I think I'd leave it ...